CondTools/DT/src/DTPerformanceHandler.cc

```diff
@@ -78,6 +78,11 @@ void DTPerformanceHandler::getNewObjects() {
   std::map<std::string, popcon::PayloadIOV>::iterator itag =
     mp.find( dataTag );
 */
+  cond::Time_t snc = runNumber;
```

Comment:
@iarspider , isn't clang analyzer going to complain that `snc` is not used when `runNumber <= last` ? I think this assignment is not needed and may be you can just directly add it to `m_to_transfer` below